target/linux/ipq40xx/base-files/etc/board.d/01_leds

```diff
@@ -40,6 +40,12 @@ netgear,ex6150v2)
 meraki,mr33)
 	ucidef_set_interface_lan "eth0"
 	;;
+qxwlan,e2600ac-c1|\
+qxwlan,e2600ac-c2)
+	ucidef_set_led_wlan "wlan2g" "WLAN0" "e2600ac:green:wlan0" "phy0tpt"
+	ucidef_set_led_wlan "wlan5g" "WLAN1" "e2600ac:green:wlan1" "phy1tpt"
+	ucidef_set_led_usbport "usb" "USB" "e2600ac:green:usb" "usb1-port1" "usb2-port1" "usb3-port1" "usb4-port1"
```

Comment:
If you want, you can convert to the DeviceTree based usb led port trigger 

https://github.com/openwrt/openwrt/commit/ef5ff08662532a6b1770214569e4c27b05e1dc35

Reply:
DT based one should be used as userspace shell is for filling the gaps only and should be kept to a minimum.

Reply:
> Since 4.19 support is also added for ipq40xx, can you also add the needed DTS and dtb patch for 4.19?

yeah, that would be sweet.

If you do, please drop the "qcom,ipq4019" compatible from the .dts in files-4.19. Upstream don't want them see: a2501ebfb06688b2519cd06583c0468522257f3b.

Reply:
My fork is an old version of openwrt, there is no 4.19 in the source. Do I need to re-pull one?

Reply:
Yes, rebase on master please.

Reply:
This pull requests, do I need to delete it?

Reply:
> This pull requests, do I need to delete it?

I think something got lost in translation. But oh well. 
No, you don't need to delete the/this pull request. As far as the usb port led trigger is concerned either method was ok, but it's nice that you converted it. :+1: